meta.go

```diff
@@ -272,6 +274,36 @@ func (t *TermID) Validate() error {
 	return nil
 }
 
+// String returns the term in it's canonical string format.
+// This would be one of:
+//   tenant:owner/name/revision
+//   owner/name/revision
+//   owner/name
+//   name/revision
+//   name
+func (t *TermsId) String() string {
+	id := ""
+	if t.Tenant != "" {
+		id = fmt.Sprintf("%s:", t.Tenant)
+	}
+	if t.Owner != "" && id == "" {
+		id = t.Owner
+	}
+	if id == "" {
+		id = t.Name
+	} else {
+		if t.Owner == "" {
+			id = fmt.Sprintf("%s%s", id, t.Name)
```

Comment:
If owner is blank don't we expect id to still be blank? does this line ever do anything? or is tenant:name a valid id. If it is can you list it above?